package.json

```diff
@@ -6,6 +6,8 @@
   "scripts": {
     "deploy:contracts:rinkeby": "npx hardhat deploy --network rinkeby --tags Export,$npm_config_tags",
     "deploy:contracts:mainnet": "npx hardhat deploy --network mainnet --tags Export,$npm_config_tags",
+    "export:contracts:rinkeby": "npx hardhat deploy --network rinkeby --tags Export",
```

Comment:
It seems like a very strange approach to me, it's `hardhat deploy` which doesn't deploy anything...
That works and it's Ok, but I would suggest to research possible ways to do it using some other command.

Reply:
Yea, I agree this is a bit odd. In general, what I wanted to achieve is that when we deploy the export part happens automatically, so that we don't run danger forgetting it. For that reason, in principle we will never have to use these scripts, because exporting runs after each deploy. For this PR, I just needed to run this command, to export the contracts once, without deploying anything and found it useful. So I can delete them alltogether.